Src/docker-compose.yml

```diff
@@ -36,7 +36,7 @@
         dockerfile: DeUrgenta.RecurringJobs.Dockerfile
       environment:
         - ASPNETCORE_URLS=http://+:80
-        - ConnectionStrings__DbConnectionString=Server=postgres;Port=5432;Database=de-urgenta-db;User Id=docker;Password=docker;
+        - ConnectionStrings__DbConnectionString=${DB_CONNECTION_STRING}
```

Comment:
I don't think this is a correct syntax. it's either 

`VAR: ${OTHER_VAR_FROM_ENV}`

or 

`- VAR=VALUE`

But not both at the same time. If you want the DB connection string to come from the `.env` file, all the vars in the `environment` settings for the API should be read from the `.env` file.

Reply:
I saw is that you should not have mixed syntax for the same service
You might be able to mix them up for distinct services

Reply:
The syntax is valid. Both `: ${BLAH}` and `= ${BLAH}` work fine.

Reply:
we ❤️ consistency: everyone now has `key: ${env_variable}`